conf/messages.en

```diff
@@ -337,9 +337,9 @@ memberDetailsCYA.viewOnly.link = View previous version of member’s details (op
 psaDeclaration.title = Declaration
 psaDeclaration.heading = Declaration
 psaDeclaration.paragraph = By continuing you declare that:
-psaDeclaration.listItem1 = you are submitting this tax return in your capacity as scheme administrator
-psaDeclaration.listItem2 = the information you have provided in this tax return is correct and complete to the best of your knowledge and belief
-psaDeclaration.listItem3 = you understand that if you have made a false statement in this tax return, you may be liable to a penalty, and that false statements may also lead to prosecution
+psaDeclaration.listItem1 = you are submitting this return in your capacity as scheme administrator
```

Comment:
@joshhumphreys this PR was sitting here too long, it become stale and have some conflicts. (main code moved on since this was created)
Could you please have a look and either resolve the conflicts or re-do the changes on a fresh branch?